src/frontend/js/data/frontEndData.ts

```diff
@@ -0,0 +1,12 @@
+import { handle } from '../utils/errors/handle';
+import { CommonDataProps } from '../types/commonDataProps';
+
+export const appFrontendContext: CommonDataProps['context'] = (window as any)
+  .__ashley_frontend_context__?.context;
+
+try {
+  if (!appFrontendContext)
+    throw new Error('No context frontend context available');
```

Comment:
I don't see the interest to throw an error then catching it and then handle the error. Why not directly call `handle` ?
```suggestion
    handle(new Error('No context frontend context available'));
```

And then remove the `try... catch...` block